pkg/provision/mbus/api.go

```diff
@@ -85,6 +88,42 @@ func (n *Deployments) listPublicIps(ctx context.Context, _ []byte) (interface{},
 	return ips, nil
 }
 
+type nameArgs struct {
```

Comment:
shouldn't that be `listPrivateIPsArgs` ?

Reply:
it was done like this for consistency with the other [id](https://github.com/threefoldtech/zos/blob/main/pkg/provision/mbus/reservation.go#L16) input struct, but sure will change it